config_spec.yaml

```diff
@@ -23,6 +23,41 @@ accept-keysend:
   description: |
     Allow others to send payments directly to your public key through keysend instead of having to get a new invoice
   default: false
+accept-amp:
+  type: boolean
+  name: Accept Spontaneous AMPs
+  description: |
+    If true, spontaneous payments through AMP will be accepted. Payments to AMP
+    invoices will be accepted regardless of this setting.
+  default: false
+reject-htlc:
+  type: boolean
+  name: Reject Routing Requests
+  description: |
+    If true, LND will not forward any HTLCs that are meant as onward payments. This option will still allow LND to send
+    HTLCs and receive HTLCs but lnd won't be used as a hop.
+  default: false
+min-chan-size:
+  type: number
+  name: Minimum Channel Size
+  description: |
+    The smallest channel size (in satoshis) that we should accept. Incoming channels smaller than this will be rejected.
```

Comment:
remove `(in satoshis)` --> this is repetitive in the UI as the unit value is displayed as satoshis